cmd/utils/flags.go

```diff
@@ -576,29 +629,32 @@ var (
 	}
 
 	// ATM the url is left to the user and deployment to
-	JSpathFlag = cli.StringFlag{
+	JSpathFlag = DirectoryFlag{
 		Name:  "jspath",
 		Usage: "JavaScript root path for `loadScript`",
-		Value: ".",
+		Value: DirectoryString("."),
 	}
 
-	WhisperEnabledFlag = cli.BoolFlag{
-		Name:  "shh",
-		Usage: "Enable Whisper",
+	// Gas price oracle settings
+	GpoBlocksFlag = cli.IntFlag{
```

Comment:
should we remove all the gas price oracle flags? We are not using this, right?

Reply:
I've commented them out. I don't want to fully remove them yet.